### ISSUE TITLE
hotkeys using alt/option not working for me

### ISSUE THREAD

**Issue by mbkupfer** [zserge/dotfiles#1]
hi @zserge 

I saw your dotfiles from a hackernews articles and wanted to port them over. They unfortunately don't work for me. Any reason why?

I'm on a mac os, using iterm2.

**Comment by globalshrug** [comment]
> hi @zserge
> 
> I saw your dotfiles from a hackernews articles and wanted to port them over. They unfortunately don't work for me. Any reason why?
> 
> I'm on a mac os, using iterm2.

You probably need to deal with your option key mapping to mod .. there is an option for it in iTerm2 somewhere. I'm on Kitty now and haven't used iTerm2 in a long time. Can confirm that the tmux config works fine for me.

**Comment by vorband** [comment]
you can change it in the Profiles/Keys setting in iterm2 for either left or right option key (Normal/Meta/Esc+).

Mind that depending on your keyboard layout some keys (e.g. ~ or @ on a German keyboard) require the option key. so maybe changing the behavior of only one of them (left or right) makes sense.

**Comment by mbkupfer** [comment]
@zserge it must be a bug in iterm2, but it appears that the meta keys were getting crossed (right was sending left and vis-a-versa). I also see that it isn't allowing me to send both meta keys...odd stuff. Anyways, I got it to work now. Here is the issue I found in case you were curious: https://gitlab.com/gnachman/iterm2/-/issues/8436

@ashleyharvey haven't heard of kitty till now, but I'll give it a shot. Thanks
